app/js/lib/common/errors.js

```diff
@@ -0,0 +1,8 @@
+export function HttpError(statusCode, message) {
```

Comment:
That looks overkilling because we don't define any specific Error in the app so far, which we should to make it easier to at least debug. The only way to make an error look nice in the console is to change it's `.name` property. That alone is useful.
But being able to distinguish properly between errors is also very useful. (what this PR does).

As an example here is what we did in the server: https://github.com/fxbox/calendar-server/blob/master/utils/errors.js
I feel strongly that it's useful (even more in the server because it sets the HTTP status code too).

Starting a quite dummy file makes defining new errors a lot easier. :)

That said, it's useful to have this in a separate file: I'm importing it in 2 different files already: 1 to throw it, 1 to check it.